### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
Add support for Python 3.8 and 3.9.

### CODE REVIEW

File: src/z3c/breadcrumb/tests.py [hunk 1]
```diff
@@ -51,24 +51,26 @@ def doctest_GenericBreadcrumb_interface():
         >>> verifyObject(interfaces.IBreadcrumb, breadcrumb)
         True
 
-    """
+    """  # noqa: F821 undefined name 'rootFolder'
```
Comment (mgedmin) [inline src/z3c/breadcrumb/tests.py:54]: flake8 allows you to say what extra names get put into doctest globals via the setUp.  You do this by adding

```
[flake8]
builtins = rootFolder
```

This should be doable via [flake8] additional-config in .meta.toml?

**Review by mgedmin — APPROVED** [review]
LGTM

File: src/z3c/breadcrumb/tests.py (hunk 1 above)
Comment (icemac) [reply src/z3c/breadcrumb/tests.py]: Fixed in https://github.com/zopefoundation/z3c.breadcrumb/pull/2/commits/6eac84f495373dbf07f84b4ae6af634e9dfbb165.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**